eclair-core/src/main/scala/fr/acinq/eclair/NodeParams.scala

```diff
@@ -170,11 +173,13 @@ object NodeParams {
     val nodeAlias = config.getString("node-alias")
     require(nodeAlias.getBytes("UTF-8").length <= 32, "invalid alias, too long (max allowed 32 bytes)")
 
-    val overrideFeatures: Map[PublicKey, (ByteVector, ByteVector)] = config.getConfigList("override-features").map { e =>
+    val features = ByteVector.fromValidHex(config.getString("features"))
+    Features.validateFeatureGraph(features).foreach(e => throw e)
```

Comment:
I don't like that we  break from the `require` pattern, but can't see a better way of doing this right off the bat.

Reply:
Yeah me too, I'll improve that.